google-cloud-storage/src/test/java/com/google/cloud/storage/BucketInfoTest.java

```diff
@@ -56,6 +56,7 @@
   private static final User OWNER = new User("user@gmail.com");
   private static final String SELF_LINK = "http://storage/b/n";
   private static final Long CREATE_TIME = System.currentTimeMillis();
+  private static final Long UPDATE_TIME = CREATE_TIME - 1L;
```

Comment:
Why `CREATE_TIME - 1L` not just `CREATE_TIME`?

Reply:
we could do that as well but for the different value i would prefer this and one more thing these is only for testing purpose.

Reply:
If the update time of an object differs from creating time, it means that the object has been changed since creating.

Reply:
done. @dmitry-fa PTAL